process_report/process_report.py

```diff
@@ -53,8 +60,9 @@ def main():
 
     projects = list(set(projects + timed_projects_list))
 
-    remove_non_billables(merged_dataframe, pi, projects, args.output_file)
+    billable_projects = remove_non_billables(merged_dataframe, pi, projects, args.output_file)
     remove_billables(merged_dataframe, pi, projects, "non_billable.csv")
+    export_pi_billables(billable_projects, args.output_folder)
```

Comment:
can we also make this optional and triggered by a flag? I think the default behavior should still be to produce the 2 reports i.e. billable and non-billable, but the user can say `--export-by-pi` and then it should split it out into multiple files. @joachimweyl thoughts?

Reply:
that makes sense, as long as it is a flag we know to use and use every month until we decide to be done with it.

Reply:
@QuanMPhm pointed out that the billable csv is still written, so I am okay with the current behavior then and don't care much about the flag.

Reply:
@naved001 @joachimweyl It should only take minutes for me to make it optional for the script to export the invoice csv by pi or as combined or as both. I'll do what you prefer.